docs/gettingstarted/modfiles.md

```diff
@@ -248,3 +248,39 @@ An entry in `neoforge.mods.toml` does not need a corresponding `@Mod` annotation
 [versioning]: versioning.md
 [enumextension]: ../advanced/extensibleenums.md
 [featureflags]: ../advanced/featureflags.md
+## Mod files tree
+```
+root
+ |-- .gradle
+ |-- gradle
+ |-- build
+ |-- run
+ |-- src
+ |    |-- main
+ |    |    |-- java
+ |    |    |    |-- <com/net>
+ |    |    |    |    |-- <username>
+ |    |    |    |    |    |-- <modid>
+ |    |    |    |    |    |    |-- <some packages and single files>
+ |    |    |    |    |    |    |-- <modname>.java
+ |    |    |    |    |    |    |-- config.java
+ |    |    |-- resources
+ |    |    |    |-- assets
+ |    |    |    |    |-- <modid>
+ |    |    |    |-- data
+ |    |    |    |    |-- minecraft
+ |    |    |    |    |-- <modid>
+ |    |-- test
+ |-- build.gradle
+ |-- gradle.properties
+ |-- gradlew   (file)
+ |-- gradlew.bat
+ |-- settings.gradle
+```
+The `gradle` and `.gradle` folders are Gradle source folders (of the ModDevGradle plugin) that you will not normally need to modify.
```

Comment:
No? The gradle documentation tells you exactly what these folders are for: https://docs.gradle.org/current/userguide/gradle_directories.html#project_root